doc/nrf/ug_thread_power_consumption.rst

```diff
@@ -0,0 +1,102 @@
+.. _thread_power_consumption:
+
+OpenThread power consumption
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page provides information about the amount of power used by Sleep End Devices (SEDs) and Synchronized Sleep End Devices (SSEDs) using OpenThread stack.
+
+.. _thread_power_consumption_methodology:
+
+Measurements methodology
+************************
+
+The measurement setup consists of:
+
+* An nrf52840dk_nrf52840 board used as Thread leader.
+* A DUT board used as Thread child.
+* A `Power Profiler Kit II (PPK2)`_ attached to the DUT according to its "Quick start" guide.
+
+The leader board is flashed with the regular :ref:`Thread CLI sample <ot_cli_sample>` firmware.
+The DUT board is flashed with the :ref:`Thread CLI sample low power mode <ot_cli_sample_low_power>` firmware.
+
+After the Thread network is enabled on the leader, the child is configured with the desired parameters.
+Once attached to the network the child device will switch to sleep mode (disabling all peripherals) and wake up according to the configured period only.
```

Comment:
```suggestion
Once attached to the network, the child device will switch to sleep mode (disabling all peripherals) and wake up according to the configured period only.
```
Would "schedule" work here instead of "period"?
Alternatively, it could just "...according to its configuration:" and have the next two sentences as bullet points.